src/discordcr/client.cr

```diff
@@ -369,6 +392,7 @@ module Discord
 
     private def handle_reconnect
       # Close the websocket - the reconnection logic will kick in
+      @instant_reconnect = true
```

Comment:
Instead of setting `@instant_reconnect`? That way it will still wait one second before reconnecting due to op7, which is what this is trying to avoid.